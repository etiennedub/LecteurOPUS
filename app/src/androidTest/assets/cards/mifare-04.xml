<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">0426CA60</List><List index="1">A2503181</List><List index="2">4248FA00</List><List index="3">0000000F</List><List index="4">00000000</List><List index="5">30F50523</List><List index="6">00000000</List><List index="7">07D60151</List><List index="8">01C7580C</List><List index="9">8240351F</List><List index="10">40000000</List><List index="11">22284541</List><List index="12">01C7580C</List><List index="13">80C05604</List><List index="14">49000000</List><List index="15">20281C6A</List></Pages></Mifare>