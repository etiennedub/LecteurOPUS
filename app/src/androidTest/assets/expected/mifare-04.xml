<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="12/12/2016" Time="17:22" Transfert="true" OperatorId="20" BusId="3" OperatorName="RTC" BusName="LEBUS 4" /><List index="1" Date="12/12/2016" Time="17:10" Transfert="false" OperatorId="20" BusId="5" OperatorName="RTC" BusName="" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>
